dotCMS/src/main/java/com/dotmarketing/db/DockerSecretsUtil.java

```diff
@@ -0,0 +1,100 @@
+package com.dotmarketing.db;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Original code was taken from https://github.com/carsdotcom/docker-secrets-java/blob/master/src/main/java/com/cars/framework/secrets/DockerSecrets.java
+ * and modified to our needs (License type: Apache 2.0)
+ * @author github.com/carsdotcom
+ */
+public class DockerSecretsUtil {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)